test/integration/functional_test.go

```diff
@@ -698,7 +698,7 @@ func validateSSHCmd(ctx context.Context, t *testing.T, profile string) {
 	if NoneDriver() {
 		t.Skipf("skipping: ssh unsupported by none")
 	}
-	want := "hello\r\n"
```

Comment:
rather than fixing the test, I rather find out why this test started to fail, are we using the wrong SSH client ?
could we either add a test to double check we are using the correct SSH client
(we have two options, native client and libmachine client...)

Reply:
FWIW, both options should yield the same output. 

I'd like to get a root cause, but I believe this PR is now doing the right thing (see my last comment).